#coordinates of starting point
float32 x_p1
float32 y_p1
float32 z_p1

#coordinates of mouth
float32 x_p2
float32 y_p2
float32 z_p2
